spec/rhc/commands/snapshot_spec.rb

```diff
@@ -38,6 +38,19 @@
       it { run_output.should_not match 'some save output' }
     end
 
+    context 'when saving a snapshot into existing file' do
+      let(:arguments) {['snapshot', 'save', '--app', 'mockapp', '-f', 'mockapp.tar.gz']}
+      before do
+        File.stub(:exists?).and_return(true)
+        RHC::Helpers.should receive(:agree).with(/Do you want to overwrite this file?/).and_return(true)
+        subject.class.any_instance.should_receive(:exec).with(/ssh #{@ssh_uri.user}@#{@ssh_uri.host} 'snapshot' > #{@app.name}.tar.gz/).and_return([0, 'some save output'])
```

Comment:
Why is it `#{@app.name}.tar.gz` here when you use `mockapp.tar.gz` in the arguments array?